src/mappers/dissolution/dissolutionRequest.mapper.ts

```diff
@@ -0,0 +1,20 @@
+import 'reflect-metadata'
+
+import { provide } from 'inversify-binding-decorators'
+
+import { DirectorRequest, DissolutionCreateRequest } from 'app/models/dto/dissolutionCreateRequest'
+import DirectorToSign from 'app/models/session/directorToSign.model'
+
+
+@provide(DissolutionRequestMapper)
+export default class DissolutionRequestMapper {
+
+  public mapToDissolutionRequest(directors: DirectorToSign[]): DissolutionCreateRequest {
+    const dtoDirectors: DirectorRequest[] = directors.map(d => {
+      return {name: d.name, email: d.email!, onBehalfName: d.onBehalfName}
+    })
+    return {
+      directors: dtoDirectors
+    }
+  }
+}
```

Comment:
I think this can be tidied up to:
```
return {
  directors: directors.map(this.mapToDirectorRequest)
}

...

private mapToDirectorRequest(director: DirectorToSign): DirectorRequest {
  return {
    name: director.name
    email: director.email,
    on_behalf_name: director.onBehalfName
  }
}
```